libnavui-androidauto/src/main/java/com/mapbox/androidauto/internal/surfacelayer/GLES20LayerHost.kt

```diff
@@ -0,0 +1,32 @@
+package com.mapbox.androidauto.internal.surfacelayer
+
+import android.opengl.GLES20
+import com.mapbox.androidauto.internal.logAndroidAutoFailure
+import com.mapbox.maps.CustomLayerHost
+import com.mapbox.maps.CustomLayerRenderParameters
+
+// This layer fixes an issue on some Xiaomi devices that causes bitmap widgets not to render
+class GLES20LayerHost : CustomLayerHost {
+
+    private var program = 0
+
+    override fun initialize() {
+        // Setup the OpenGL ES Program
+        program = GLES20.glCreateProgram()
```

Comment:
When testing, removing this does not make a difference. It works without creating a program

but it fails if you remove addPersistentStyleCustomLayer(GLES20LayerHost)

Reply:
Indeed, somehow I forgot to check that.